euth_wagtail/assets/scss/components/_projects.scss

```diff
@@ -155,7 +158,22 @@
 }
 
 .project-tile-list {
-    @include rem(padding, 50px 40px);
+    @include rem(padding, 30px 30px 30px 0);
+
+    .col-md-9,
+    .col-xs-11 {
+        padding-left: 0;
+    }
+
+    .col-md-1,
+    .col-xs-1 {
+        @include rem(padding-right, 10px);
+        @include rem(padding-left, 0);
+    }
+}
+
+.project-tile-badges {
+    @include rem(margin-top, 10px);
 }
 
 .badge-group {
```

Comment:
Can you move this guy to the new `badges.scss` you created? I think it fits there better now, that we have this file.